.github/workflows/nightly-upload.yml

```diff
@@ -1,9 +1,10 @@
 name: Upload docs to production
 
 on:
-  schedule:
-    # UTC timezone
-    - cron: '0 6 * * *'
+  push
+  # schedule:
+  #   # UTC timezone
+  #   - cron: '0 6 * * *'
```

Comment:
Don't forget to revert this back to `schedule` before merging :grin:

Reply:
Reverted